tutorials/applications/Spin chain of 3 atoms in XY mode.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Please include some Introductory text so that people know the goals of the tutorial.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>